src/openpersonen/api/models.py

```diff
@@ -1,37 +1,69 @@
+import base64
 import uuid
+from datetime import timedelta
 
-from django.conf import settings
+from django.db import models
 from django.template import loader
 from django.utils import dateformat, timezone
+from django.utils.translation import ugettext_lazy as _
 
 import requests
+from solo.models import SingletonModel
 
 
-class Client:
+class StufBGClient(SingletonModel):
+
+    ontvanger_organisatie = models.CharField(_("organisatie"), max_length=200)
+    ontvanger_administratie = models.CharField(_("administratie"), max_length=200)
+    ontvanger_applicatie = models.CharField(_("applicatie"), max_length=200)
+    ontvanger_gebruiker = models.CharField(_("gebruiker"), max_length=200)
+    zender_organisatie = models.CharField(_("organisatie"), max_length=200)
+    zender_administratie = models.CharField(_("administratie"), max_length=200)
+    zender_applicatie = models.CharField(_("applicatie"), max_length=200)
+    zender_gebruiker = models.CharField(_("gebruiker"), max_length=200)
+    url = models.URLField(_("url"))
+    user = models.CharField(_("user"), max_length=200)
+    password = models.CharField(_("password"), max_length=200)
+
+    class Meta:
+        verbose_name = _("Stuf BG Client")
+
+    def _get_headers(self):
+        credentials = (self.user + ":" + self.password).encode("utf-8")
```

Comment:
Nitpicking, but I like f-strings :p
```suggestion
        credentials = f"{self.user}:{self.password}".encode("utf-8")
```

Reply:
Hehe, since this is a Python 3 project I think f-strings are good to use :)